src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -186,6 +186,7 @@ class GoogleExtensions(
    */
   override def onGroupUpdate(groupIdentities: Seq[WorkbenchGroupIdentity], samRequestContext: SamRequestContext): Future[Unit] = {
     for {
+      start <- clock.instantNow
```

Comment:
What is the goal of the clock timing here? Just useful log messaging?

Reply:
https://github.com/broadinstitute/sam/pull/535/files#r672506030